newlib/libc/tinystdio/bufio.c

```diff
@@ -149,9 +149,6 @@ __bufio_put(char c, FILE *f)
 	return ret;
 }
 
-extern FILE *const stdin _ATTRIBUTE((__weak__));
```

Comment:
This is required to avoid pulling in stdin/stdout when they are otherwise unused. If they are not used anywhere else in the code, they will be NULL; you'll note that stdout is checked for NULL before dereferencing, so that should be OK.